client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -161,6 +166,45 @@ const siteSetupFlow: Flow = {
 		const { setDesignOnSite } = useDispatch( SITE_STORE );
 		const dispatch = reduxDispatch();
 
+		const [ isLoadingLaunchpadExperiment, launchpadExperimentAssigment ] =
+			useExperiment( LAUNCHPAD_EXPERIMENT_NAME );
+
+		const getLaunchpadScreenValue = (
+			intent: string,
+			shouldSkip: boolean
+		): 'full' | 'skipped' | 'off' => {
+			if ( ! isLaunchpadIntent( intent ) || isLaunched ) {
+				return 'off';
+			}
+
+			if (
+				isLoadingLaunchpadExperiment ||
+				! launchpadExperimentAssigment?.variationName ||
+				launchpadExperimentAssigment.variationName === 'control'
+			) {
+				if ( shouldSkip ) {
+					return 'skipped';
+				}
+
+				return 'full';
+			}
+
+			if ( launchpadExperimentAssigment.variationName === 'treatment_no_launchpad' ) {
+				return 'off';
+			}
+
+			if ( launchpadExperimentAssigment.variationName === 'treatment_skip_launchpad' ) {
+				return 'skipped';
+			}
```

Comment:
I think this should be looking at `'treatment'` for the variation, and removing this old logic for the A/B/C test.
```suggestion
			if ( launchpadExperimentAssigment.variationName === 'treatment' ) {
				return 'skipped';
			}
```

Reply:
Also, it feels like this logic should also be exposed/shared with `useLaunchpadDecider()`.